### ISSUE TITLE
How to disable entirely without affecting other TS checking in VS Code

### ISSUE THREAD

**Issue by electrovir** [microsoft/typescript-styled-plugin#171]
I frequently run into `ts-styled-plugin(9999)` errors in VS Code that are entirely false negatives and show up all over the place for our devs.

I have no idea why these errors show up. I do not have ESLint, TSLint, typescript-styled-plugin, or their respective VS Code extensions installed _anywhere_ on my computer. It's not a part of my actual CLI TS compilation. It just shows up in VS Code.

I do not want this package to run at all, I want to entirely turn it off. The only way I've figured out how to do that so far is by entirely disabling typescript checking in VS Code with `"typescript.validate.enable": false` but that is not an acceptable solution as that disables _all_ typescript checking in VS Code.

How do I block all `ts-styled-plugin(9999)` errors in VS Code without entirely disabling TS checking?

**Comment by machineghost** [comment]
Seriously: a validator that gives a million false validation methods isn't a helpful tool, it's a drain on your mental resources.  As much as I'd love true styled-component CSS validation, I'll take nothing over a ton of false positives.

**Comment by machineghost** [comment]
According to the documentation, modifying `jsconfig.json` *should* have the desired effect:

```
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-styled-plugin",
        "validate": false
      }
    ]
  }
}


```

... but it doesn't work.  I added that to mine, restarted VS Code, and still saw bogus "errors" :(

**Comment by nick-kl4us** [comment]
This plugin is LAZYYYYY
how do you not support SVG attributes out of the box?
go home

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)
